frontend-react/src/components/StaticCompare.tsx

```diff
@@ -0,0 +1,143 @@
+import { ReactElement, useCallback, useEffect, useState } from "react";
+import DOMPurify from "dompurify";
+import { ScrollSync, ScrollSyncPane } from "react-scroll-sync";
+
+import { Diff, SES_TYPE } from "../utils/diff";
+import { splitOn } from "../utils/misc";
+
+interface StaticCompareProps {
+    rightText: string;
+    leftText: string;
+}
+
+const insertHighlight = (
```

Comment:
can we get a jsdoc here? i know it's not exported, but it's somewhat complex and the arg names are not super illuminating as to what's being processed

Reply:
Will do. My next PR (which will be ready later today) reworks this whole highlighting approach to better deal with structured json versus unstructured text.